src/qat/purr/backends/qiskit_simulator.py

```diff
@@ -29,12 +30,33 @@
 log = get_default_logger()
 
 
+class QatMPSConfig(BaseSettings):
+    """
+    The default settings for using the MPS backend in the Qiskit Simulator.
+    """
+
+    model_config = SettingsConfigDict(env_previx="QAT_MPS_")
+    MAX_BOND_DIMENSION: int = 128
+    """Default maximum bond dimension for MPS simulations."""
+    TRUNCATION: float = 1e-12
+    """The error threshold for dynamically truncating the bond dimension of MPS."""
+
+
+qatmpsconfig = QatMPSConfig()
+
```

Comment:
Done, let me know if this is what you had in mind.

Had to move the `QatMPSConfig` class to avoid circular imports. Could put it in a new file, but seemed simpler to me to just add it to `QatConfig`.

Reply:
Looks good